src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -770,7 +774,7 @@ private function copyOrMove($uuid, $destParentUuid, $userId, $webspaceKey, $loca
 
         $this->documentManager->find($document->getUuid(), $originalLocale);
 
-        return $this->documentToStructure($document);
+        return $this->documentToStructure($originalDocument);
```

Comment:
no also before the "original" was returned. that was done by calling find with the original locale right before this line.